src/rules/de.toml

```diff
@@ -32,6 +32,8 @@ replacements = [
   ["Abb.", "Abbildung"],
   ["geb.", "geboren"], 
   ["gegr.", "gegründet"],
+  ["vs.", "versus"],
+  ["kgl", "königlich"],
```

Comment:
Is this really occuring without a period? I wonder how many words there are that have "kgl" in the middle somewhere, which would be replaced as well 😅 nothing comes to mind right now, but might not be safe

Reply:
Oh maybe is it better with kgl. 

I found 

eidg. - Eidgenossenschaftlich

Maybe add this Zoo?

Reply:
See my comment above, same applies here. And I think the same would also apply for "eidg.". What do you think?

Reply:
> Is this really occuring without a period? I wonder how many words there are that have "kgl" in the middle somewhere, which would be replaced as well 😅 nothing comes to mind right now, but might not be safe

you are right. Not without A Period.

So "kgl." And "Eidg." Are better. 👌